MODULE.bazel

```diff
@@ -143,7 +143,6 @@ maven.install(
     fail_if_repin_required = True,
     generate_compat_repositories = True,
     lock_file = "//:contrib_rules_jvm_deps_install.json",
-    resolver = "maven",
```

Comment:
So to be clear, this switches us back to coursier as the resolver right?

Reply:
Yes, back to `coursier` now that it supports boms